src/UncontrolledNavigation.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+
+export default class UncontrolledNavigation extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { isOpen: true };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  render() {
+    return <Navigation open={this.state.isOpen} onOpen={this.toggle} {...this.props} />;
```

Comment:
What is `Navigation`? It isn't something in reactstrap and isn't even being imported.

Reply:
@TheSharpieOne and I am an idiot.  Disregard.